pkg/skaffold/config/options.go

```diff
@@ -32,6 +32,7 @@ type SkaffoldOptions struct {
 	Namespace         string
 	Watch             []string
 	WatchPollInterval int
+	PortForward       bool
```

Comment:
If you put this field next to the rest of the bool fields, it will fix the linter error.

Kind of an odd linter that we might want to remove cc @dgageot